packages/@react-aria/utils/test/mergeRefs.test.tsx

```diff
@@ -40,14 +45,18 @@ describe('mergeRefs', () => {
       let target = null;
 
       const TextField = (props) => {
-        ref1 = useRef(null);
-        ref2 = useRef(null);
+        let internalRef1 = useRef(null);
+        let internalRef2 = useRef(null);
+        useEffect(() => {
```

Comment:
other option i guess would be to exclude the compiler from running on test files, but seems better to always be in good habits